### PR TITLE
[CI] Add cron job to update nightly new translations from Transifex

### PR DESCRIPTION
**Pull request by AenBleidd**
Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

### CODE REVIEW

**Comment by AenBleidd** [comment]
@talregev, PR is ready for review

File: locale/update_translations.sh [hunk 1]
```diff
@@ -57,10 +58,9 @@ done
 
 cd ${srcdir}
 
-echo "Translations compiled successfully. Now some manual steps:"
-echo " 1. less BOINC-Manager-pofilter.txt # check output from pofilter and adjust translations then start this script again"
-echo " 2. git add -u # only update already tracked files (add new files when needed too)"
-echo " 3. git commit -m \"Locale: Update localization files [skip ci]\""
-echo " 4. git push"
+git add .
```
Comment (talregev) [inline locale/update_translations.sh:61]: After it git add, what happen? The code is not updated. Why you delete git push?

File: locale/update_translations.sh [hunk 2]
```diff
@@ -57,10 +58,9 @@ done
...
+git diff-index --cached --quiet HEAD || { git commit -m "Locale: Update localization files"; git push origin HEAD; }
```
Comment (talregev) [inline locale/update_translations.sh:62]: Now I see the code is change here. How it push without a password?

Comment (AenBleidd) [reply locale/update_translations.sh:62]: GitHub Actions have embedded credentials, so when you do a push from the CI, you are able to push commits to the repo from which CI is run.

**Review by talregev — APPROVED** [review]
I didn't test it. The changes seems ok. We should test it.

**Comment by AenBleidd** [comment]
After careful consideration and offline discussion with @CharlieFenton, I decided not to continue with this functionality and decided to keep this a part of Release process. Closing this for now.

**Pull request closed without merging**